### PR TITLE
DEVX-2164: Help with ccloud-stack adoption by showing the CLI commands that are executed

### PR DESCRIPTION
**Pull request by ybyzek**
### Description 

https://confluentinc.atlassian.net/browse/DEVX-2164

- show precisely which ccloud commands are run
- give indication of what is free and what incurs charges

### Author Validation

Waiting for docs migration to complete to verify local build of this PR

- [x] Documentation

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

- [x] Documentation

### CODE REVIEW

**Comment by ybyzek** [comment]
@caleb-grillo @bechols -- can you please review the PR to ensure the statements about cost/billing are valid?

File: ccloud/docs/ccloud-stack.rst [hunk 1]
```diff
@@ -30,8 +30,32 @@ Caution
 =======
 
 This utility uses real |ccloud| resources.
+If you just run ``ccloud-stack`` without enabling |ccloud| ksqlDB and do not create any topics or other resources, then there is no billing charge.
```
Comment (rspurgeon) [inline ccloud/docs/ccloud-stack.rst]: IMO we remove this.  We don't want users to use the utility unless it's for something useful and we don't want them to create resources (even logical non-cost associated ones) for no reason.

Comment (ybyzek) [reply ccloud/docs/ccloud-stack.rst]: There is still immense value in running ``ccloud-stack`` without ksqlDB because it properly creates the credentials for each resource (and wildcard ACLs).  This addresses some user friction we have observed from new users.  The key with this PR was to be transparent about what the cost is, to encourage users to use ``ccloud-stack``.  It's a value add to say it's free, and the outcome is proper credentials.

Comment (rspurgeon) [reply ccloud/docs/ccloud-stack.rst]: >and do not create any topics or other resources, then there is no billing charge.

Can you clarify this portion of the sentence as maybe I'm not understanding what you're implying with this?

Comment (rspurgeon) [reply ccloud/docs/ccloud-stack.rst]: If you're implying that users should/could use ccloud-stack to create environments and credentials (non-billable) for _later_ use in contexts that _are_ billable, I think the message is potentially confusing.  Nothing about a new enviornment has value until a user creates a topic pushes data, which comes with a cost.  I think you could reword this to better convey what I think you're trying to say which is, you can use this to "setup" an environment with proper credentials for use later.  Regardless, the "later use" will result in a charge.

Comment (ybyzek) [reply ccloud/docs/ccloud-stack.rst]: @rspurgeon what do you think about the revised version?

Comment (rspurgeon) [reply ccloud/docs/ccloud-stack.rst]: Thanks, it's clearer for sure IMO.  WDYT about something like (I think I'm interpreting you correctly with this):

> ccloud-stack can be useful to "setup" an environment for other demos or use cases you may require by creating an environment and credentials for you.  You will only incur charges as you create topics and produce and consume data.

or something like that...

File: ccloud/docs/ccloud-stack.rst [hunk 2]
```diff
@@ -30,8 +30,33 @@ Caution
 =======
 
 This utility uses real |ccloud| resources.
+It is intended to be a quick way to create resources in |ccloud| with correct credentials and permissions, useful as a starting point from which you can then use for learning, extending, and building other examples.
+If you just run ``ccloud-stack`` without enabling |ccloud| ksqlDB, then there is no billing charge (but as soon as you produce data to the |ak| cluster or provision any other fully-managed services, there may be charges).
+If you run ``ccloud-stack`` with enabling |ccloud| ksqlDB, then you will begin in accrue charges.
```
Comment (rspurgeon) [inline ccloud/docs/ccloud-stack.rst]:
```suggestion
If you run ``ccloud-stack`` with enabling |ccloud| ksqlDB, then you will begin to accrue charges.
```

**Review by rspurgeon — APPROVED** [review]
LGTM, thanks

**Comment by bechols** [comment]
Yep, shouldn't be any costs without ksqlDB since this is creating a Basic cluster which doesn't have an uptime charge and not actually using the cluster at all.

**Review by caleb-grillo — APPROVED** [review]

File: ccloud/docs/ccloud-stack.rst [hunk 3]
```diff
@@ -30,8 +30,33 @@ Caution
 =======
 
 This utility uses real |ccloud| resources.
+It is intended to be a quick way to create resources in |ccloud| with correct credentials and permissions, useful as a starting point from which you can then use for learning, extending, and building other examples.
+If you just run ``ccloud-stack`` without enabling |ccloud| ksqlDB, then there is no billing charge until you produce data to the |ak| cluster or provision any other fully-managed services.
+If you run ``ccloud-stack`` with enabling |ccloud| ksqlDB, then you will begin to accrue charges immediately.
```
Comment (matt-mangia) [inline ccloud/docs/ccloud-stack.rst:35]: These 3 lines are great!

**Review by matt-mangia — APPROVED** [review]
I think this is super helpful & clarifies a lot for me in using the tool! Thank you :)

File: ccloud/docs/ccloud-stack.rst (hunk 3 above)
Comment (ybyzek) [reply ccloud/docs/ccloud-stack.rst:35]: Excellent, thanks for reviewing @matt-mangia

**Review by rspurgeon — APPROVED** [review]
LGTM
